frontend/src/app/pages/exercises/exercise/shared/transfer-overview/transfer-time-input/transfer-time-input.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="btn-group btn-group-sm text-nowrap">
+    <button
+        class="btn btn-small btn-outline-secondary"
+        title="Um 5 Minuten verkürzen"
+        (click)="addTransferTime(-5 * 60 * 1000)"
+    >
+        -5min
+    </button>
+    <button
+        class="btn btn-small btn-outline-secondary"
+        title="Um 1 Minute verkürzen"
+        (click)="addTransferTime(-60 * 1000)"
+    >
+        -1min
+    </button>
+    <button
+        class="btn btn-small btn-outline-secondary font-monospace fw-bold"
+        [class.text-dark]="!transfer.isPaused"
+        (click)="togglePauseTransfer()"
+        [title]="'Transfer ist ' + (transfer.isPaused ? 'pausiert' : 'aktiv')"
+    >
+        <i [ngClass]="transfer.isPaused ? 'bi-pause-fill' : 'bi-play-fill'"></i>
+        <!-- TODO: `??` would be a bit better than `||`, but messes up the code highlighting -->
```

Comment:
Maybe pull the `?? 0` to the selector so that this can never happen here? Not sure whether this is better, though.

Reply:
This wouldn't help because `| async` returns `null | T` (if the stream hasn't emitted yet, the value is `null`)

Reply:
Ok, then leave it as-is.